test/unittest/unittest_nntrainer_models.cpp

```diff
@@ -266,18 +289,29 @@ class GraphWatcher {
 };
 
 void NodeWatcher::read(std::ifstream &in) {
-  expected_output.read(in);
-  expected_dx.read(in);
+  // log prints are commented on purpose
+  // std::cout << "[=======" << node->getName() << "==========]\n";
+
+  auto read_ = [&in](auto &target) {
+    // std::cout << target.getDim();
```

Comment:
cleanup couts?

Reply:
Intended, this is too descriptive to fit in ml_logd bug needed, so commented

Reply:
alright.